### ISSUE TITLE
PYTHONPATH not unsetting on deactivate. 

### ISSUE THREAD

**Issue by pnegahdar** [pnegahdar/inenv#33]

**Comment by obi1kenobi** [comment]
I think this affects all env vars. Especially problematic when `DJANGO_SETTINGS_MODULE` leaks out from one inenv into another.

**Comment by pnegahdar** [comment]
Yeah I think it happens on env switch and doesn't effect subprocess usage (e.g. gbin). If you deactivate first it should be fine (can you double check if this is true?). Maybe we should force a deactivate on every new switch? I thought it was cool do something like "inenv project" "inenv prod" to get the envvars of prod bound to project's venv. Not sure if thats generally undesirable though.

**Comment by obi1kenobi** [comment]
I think `inenv project, inenv prod` would be confusing, because I don't expect side-effects to carry over between `inenv` invocations. I'd vote against it.
